javascript-bank/bank.js

```diff
@@ -0,0 +1,39 @@
+/* exported Bank */
+
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  if (balance <= 0 || (typeof balance !== 'number')) {
+    return null;
+  }
+
+  var newBankAccount = new Account(this.nextAccountNumber, holder);
+  newBankAccount.deposit(balance);
+  this.accounts.push(newBankAccount);
+  return this.nextAccountNumber++;
+};
+
+Bank.prototype.getAccount = function (number) {
+  for (var i = 0; i < this.accounts.length; i++) {
+    if (this.accounts[i].number === number) {
+      return this.accounts[i];
+    }
+  }
+
+  return null;
+};
+
+Bank.prototype.getTotalAssets = function () {
+
+  if (this.accounts.length === 0) return 0;
```

Comment:
This conditional is unnecessary. If the length of the `accounts` array is zero, then the loop below this would stop immediately, and leave you with the same result 👍